SpywareFilter/sections/whitelist.txt

```diff
@@ -1582,11 +1569,6 @@ nnmclub.ro,nnm-club.t0r.xyz,nnm-club-me.appspot.com,nnm-club.lib,nnm-club.me,nnm
 ! https://github.com/AdguardTeam/AdguardFilters/issues/50415
 @@||intersport.peerius.com^|$domain=intersport.de
 @@||intersport.peerius.com/tracker/*.page$domain=intersport.de
-! https://github.com/AdguardTeam/AdguardFilters/issues/50052
-@@||googletagmanager.com/gtm.js$domain=incehesap.com
```

Comment:
This rules is needed.

Reply:
https://www.incehesap.com/robotuya-robolaser-beyaz-robot-supurge-turkce-dil-destegi-distributor-garantili-fiyati-50076/
If the request is blocked, you can't add the item to cart. I see it with `EasyPrivacy` too.

Reply:
Reverted the rule.